Core/Core/Serialisation/BaseObjectDeserializerV2.cs

```diff
@@ -74,7 +76,15 @@ public Base Deserialize(string rootObjectJson)
         string objId = closure.Item1;
         // pausing for getting object from the transport
         stopwatch.Stop();
-        string objJson = ReadTransport.GetObject(objId);
+        string? objJson = ReadTransport.GetObject(objId);
+
+        //TODO: We should fail loudly when a closure can't be found
+        //but adding this throw breaks blobs tests, see CNX-8541
+        // if (objJson is null)
+        // {
+        //   throw new TransportException($"Closure {objId} was not found in transport {ReadTransport}");
```

Comment:
See https://spockle.atlassian.net/browse/CNX-8541

Reply:
Interesting! @BovineOx you should have a look at this if you haven't already

Reply:
We might want to remove the commented out code, even if we plan to get there soon, just in case...

Reply:
@gjedlicska has been pinged (as the one who implemented blobs) but we've agreed it can wait until the new year.

Reply:
Happy to remove the commented out code for now, but if there aren't any objections, I'd like to keep the comment about "don't throw here (here be dragons)"

Reply:
Let's do that, it's cleaner